src/SB/Core/x/xMath3.h

```diff
@@ -125,7 +125,7 @@ void xMat3x3Mul(xMat3x3* o, const xMat3x3* a, const xMat3x3* b);
 void xMat3x3SMul(xMat3x3*, const xMat3x3*, F32);
 void xBoxFromLine(xBox& box, const xLine3& line);
 void xBoxFromRay(xBox& box, const xRay3& ray);
-void xMat3x3Identity(xMat3x3* matrix);
+void xMat3x3Identity(xMat3x3* matrix); // May come from xClimate
```

Comment:
A bit of a nit I know, but can you remove these comments (or at least reword them somehow). They don't really "come from" xClimate, but rather would have been defined here directly in this header file and just by chance the definition in xClimate's translation unit ended up being the one that the linker chose to use (maybe it was the first/last usage or something like that).

Reply:
If you want to leave a comment about this I would probably say something like
`// TODO: These functions should be inline`